test/Routes/ListRoutesCommandTest.php

```diff
@@ -0,0 +1,496 @@
+            ],
+            [
+                'yaml',
+            ],
+        ];
+    }
+
+    /**
+     * @dataProvider sortRoutingTableDataProvider
+     * @throws ReflectionException
+     */
+    public function testCanSortResults(string $sortOrder, string $expectedOutput): void
+    {
+        $routes                = [
+            new Route("/contact", new SimpleMiddleware(), ['GET'], 'contact'),
+            new Route("/", new ExpressMiddleware(), ['GET'], 'home'),
+        ];
+        $this->routeCollection = $this->createMock(RouteCollector::class);
+        $this->routeCollection
+            ->expects($this->once())
+            ->method('getRoutes')
+            ->willReturn($routes);
+
+        $this->container = $this->createMock(ContainerInterface::class);
+        $this->container
+            ->expects($this->once())
+            ->method('get')
+            ->willReturn(
+                $this->routeCollection,
+            );
+
+        /** @var ConfigLoaderInterface $configLoader */
+        $configLoader  = $this->createMock(ConfigLoaderInterface::class);
+        $this->command = new ListRoutesCommand($this->container, $configLoader);
+
+        $this->input
+            ->method('getOption')
+            ->willReturnOnConsecutiveCalls(
+                'json', // format
+                $sortOrder, // sort
+                false, // supports-method
+                false, // has-middleware
+                false, // has-name
+                false  // has-path
+            );
+        $this->output
+            ->expects($this->atMost(2))
+            ->method('writeln');
+
+        $method = $this->reflectExecuteMethod();
+
+        self::assertSame(
+            0,
+            $method->invoke(
+                $this->command,
+                $this->input,
+                $this->output
+            )
+        );
+    }
+
+    /**
+     * @return array<array-key, array<array-key,string>>
+     */
+    public function sortRoutingTableDataProvider(): array
+    {
+        // phpcs:disable Generic.Files.LineLength
+        return [
+            [
+                'name',
+                '[{"name":"contact","path":"\/contact","methods":"GET","middleware":"MezzioTest\\\\Tooling\\\\Routes\\\\Middleware\\\\SimpleMiddleware"},{"name":"home","path":"\/","methods":"GET","middleware":"MezzioTest\\\\Tooling\\\\Routes\\\\Middleware\\\\ExpressMiddleware"}]',
+            ],
+            [
+                'path',
+                '[{"name":"home","path":"\/","methods":"GET","middleware":"MezzioTest\\\\Tooling\\\\Routes\\\\Middleware\\\\ExpressMiddleware"},{"name":"contact","path":"\/contact","methods":"GET","middleware":"MezzioTest\\\\Tooling\\\\Routes\\\\Middleware\\\\SimpleMiddleware"}]',
+            ],
+        ];
+        // phpcs:enable
+    }
+
+    /**
+     * @dataProvider filterRoutingTableDataProvider
+     */
+    public function testCanFilterRoutingTable(array $filterOptions, string $expectedOutput): void
```

Comment:
```suggestion
    public function testCanFilterRoutingTable(array $filterOptions): void
```
Parameter `$expectedOutput` is not used.